### ISSUE TITLE
release the https://farmr.net config

### ISSUE THREAD

**Issue by Fexiven** [joaquimguimaraes/farmr#99]
Could you please provide the farmr.net configuration so we can host farmr local without cloud?

**Comment by joaquimguimaraes** [comment]
Server-side is not as easy to set up as it seems. There are multiple parts (PHP, Flutter Web, Node.JS notification system and MariaDB) which would require extensive documentation for someone to set up everything properly. 

Instead, I plan on simplifying everything into one repository that anyone can install and set up with a simple script. Although this will take a while.

**Comment by Fexiven** [comment]
That would be awesome.
Thanks for the info.

**Comment by joaquimguimaraes** [comment]
I'll reopen this issue and close it when server source is released

**Comment by ByronAP** [comment]
opening the web repo(s) would be most helpful since I would like to make a custom android app client

**Comment by joaquimguimaraes** [comment]
Yes that will be released, it's just that the UI code needs some tidying up
